packages/website/src/components/ProjectLearnMore.vue

```diff
@@ -1,70 +1,49 @@
 <template>
-  <strings-domain
-    v-if="project"
-    #key="{
-      website,
-      website_title,
-      playstore,
-      medium,
-      appstore,
-      ios_github,
-      android_github,
-      github
-    }"
-    :value="project"
+  <page-section
+    v-if="
+      enableAll ||
+        (!website && !playstore && !appstore) ||
+        ((playstore || appstore || website) && (ios_github || android_github || github))
+    "
   >
-    <page-section
-      v-if="
-        enableAll ||
-          (!website && !playstore && !appstore && medium) ||
```

Comment:
Document the `medium` support removal in the commit please. :smile_cat:

Reply:
@ewlsh  Documented in PR comment and commit message